src/core/tiledmesh/qgstiledmeshboundingvolume.h

```diff
@@ -87,6 +90,27 @@ class CORE_EXPORT QgsAbstractTiledMeshNodeBoundingVolume
      */
     virtual QgsAbstractGeometry *as2DGeometry( const QgsCoordinateTransform &transform = QgsCoordinateTransform(), Qgis::TransformDirection direction = Qgis::TransformDirection::Forward ) const = 0 SIP_FACTORY;
 
+    /**
+     * Sets the bounding volume's \a transform.
+     *
+     * \see transform()
+     */
+    void setTransform( const QgsMatrix4x4 &transform ) { mTransform = transform; }
+
+    /**
+     * Returns the bounding volume's transform.
+     *
+     * This represents the transformation which must be applied to all geometries from the tile
+     * in order to transform them to the dataset's coordinate reference system.
```

Comment:
I am wondering about the bounding volumes and transforms, and whether there should be any link between how the bounding volume itself is transformed, and how geometries of the tile are transformed.

My preference would be not to keep transform as a part of bounding volume, and only store the transformed bounding volume. There's also the odd case of bounding region, where the transform is not applied to the bounding volume, but it is applied to tile content...

Reply:
Ok, this is a good move. Done.